templates/repo/settings/options.tmpl

```diff
@@ -172,7 +173,8 @@
 							<td>{{$address.Address}}</td>
 							<td>{{$.locale.Tr "repo.settings.mirror_settings.direction.push"}}</td>
 							<td>{{if .LastUpdateUnix}}{{.LastUpdateUnix.AsTime}}{{else}}{{$.locale.Tr "never"}}{{end}} {{if .LastError}}<div class="ui red label tooltip" data-content="{{.LastError}}">{{$.locale.Tr "error"}}</div>{{end}}</td>
-							<td class="right aligned">
+							<td><a data-clipboard-text="{{.PublicKey}}" readonly>{{$.locale.Tr "repo.settings.mirror_settings.push_mirror.copy_public_key"}}</a></td>
```

Comment:
`readonly` on a link?
What's that supposed to achieve?
Isn't a link already `readonly`?

Reply:
Yeah seems pointless.

Reply:
Oops leftover from when this was an input.